docs/guides/tablediff.md

```diff
@@ -153,3 +153,121 @@ SQLMESH_EXAMPLE.INCREMENTAL_MODEL ONLY sample rows:
 ```
 
 The output matches, with the exception of the column labels in the `COMMON ROWS sample data differences`. The underlying table for each column is indicated by `s__` for "source" table (first table in the command's colon operator `:`) and `t__` for "target" table (second table in the command's colon operator `:`).
+
+## Diffing tables or views across gateways
```

Comment:
Can you add a screenshot of a shell output where you run a command and see what exact matches and diffs should look and feel like?

Reply:
Also had generic metrics on performance time for table size. Ex:
- 1 billion rows across snowflake and bigquery, 50 seconds, etc.

Reply:
Update: no need to add metrics for now

Reply:
I've added some sample output as text to keep with the existing theme of this page (and also text is easier to remove my name from the schema references)

Reply:
Looks Gouda